permissions/plugin-clearcase-ucm-api.yml

```diff
@@ -8,3 +8,6 @@ paths:
 developers:
 - "praqma"
 - "atombrella"
+- "notmyfault"
```

Comment:
this is the api plugin (https://github.com/jenkinsci/clearcase-ucm-api-plugin) I assume you want the other one too

Reply:
The actual plugin is unaffected by the forthcoming change.
I'll likely not stay around maintaining these plugins, I'm just here to work down the list to cut a few releases giving people the chance to update before the next LTS arrives.

Reply:
> this is the api plugin (https://github.com/jenkinsci/clearcase-ucm-api-plugin) I assume you want the other one too

Oh wow, I just read it again. Yeah, this is indeed the wrong plugin, my PR is on clearcase-ucm-plugin 🤦🏻 
I'll file a new PR to add myself there.